core/src/test/java/com/expedia/aquila/core/util/MathUtilTest.java

```diff
@@ -0,0 +1,62 @@
+package com.expedia.aquila.core.util;
+
+import org.junit.rules.ExpectedException;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+
+public class MathUtilTest {
+
+    @Rule public final ExpectedException thrown = ExpectedException.none();
+
+    @Test
+    public void incrAndLog() {
+      final double[] data = {1.0};
+
+      assertArrayEquals({0.6931471805599453}, MathUtil.incrAndLog(data), 0.0);
+    }
+
+    @Test
+    public void exp() {
+      final double[] data = {1.0};
+
+      assertArrayEquals({2.718281828459045}, MathUtil.exp(data), 0.0);
```

Comment:
Does `assertArrayEquals({Math.E}, MathUtil.exp(data), 0.0);` pass?

Reply:
I made a mistake, it should have been `new double[]{Math.E}`, but yes it passes.